### ISSUE TITLE
support for .info

### ISSUE THREAD

**Issue by jreback** [dask/dask#978]
I *think* this should be doable lazily (maybe forget the nan count), just some summarizing data

DataFrame.info
```
<class 'pandas.core.frame.DataFrame'>
Int64Index: 4026195 entries, 0 to 567
Data columns (total 3 columns):
Date     datetime64[ns]
Close    float64
stock    object
dtypes: datetime64[ns](1), float64(1), object(1)
memory usage: 122.9+ MB
```

for sure you know dtypes & lengths of the chunks.

**Comment by mrocklin** [comment]
Would you rather have this be immediate but not have memory usage, numbers, etc. or would you rather it be slow and have all of the information?

**Comment by jreback** [comment]
already have .dtypes which iirc is immediate

I think should show full days (but maybe have same defaults - iow don't go deep on memory\usage)

**Comment by mrocklin** [comment]
I'm confused.  Do you want this to be fast but not show all of the numbers we can get by computing everything or do you want this to go slow and get you all of the numbers.  This is either going to take 1ms and have nothing but dtype and column information or take as much time as it takes to do the entire computation.

**Comment by jreback** [comment]
oh right

I would make the default fast with lazy=False ? (and then allow lazy=true for full compute
not sure what your spelling is

**Comment by jcrist** [comment]
Closed by #1213.
